helpers/azure-storage.js

```diff
@@ -6,23 +6,38 @@ const { AZURE: { account_key, account_name, container_name, sasExpiryTime } } =
 
 const blobService = azure.createBlobService(account_name, account_key);
 
+const StorageService = require('./cloudStorageServices/index');
+
 const getBlobReadStream = ({ filePath }) => {
     return blobService.createReadStream(container_name, filePath);
 }
 
 const checkIfBlobExists = ({ container = container_name, filePath }) => {
     return new Promise((resolve, reject) => {
-        blobService.doesBlobExist(container, filePath, (error, success) => {
-            if (error) {
-                return reject(error);
-            }
+        /**
+         * @deprecated
+         */
+        // blobService.doesBlobExist(container, filePath, (error, success) => {
+        //     if (error) {
+        //         return reject(error);
+        //     }
 
-            if (success && !success.exists) {
-                return reject(false);
-            }
+        //     if (success && !success.exists) {
+        //         return reject(false);
+        //     }
+
+        //     resolve(success);
+        // })
+        StorageService.CLOUD_CLIENT.fileExists(container, filePath, (error, response) => {
+          if (error) {
+            return reject(error);
+          }
 
-            resolve(success);
-        })
+          if (_.get(response, 'exists')) {
```

Comment:
@manjudr 
We can directly refer `response.exists`
It is the best practice to use null check
- What if response object does not sent exists key, an exception will be thrown `Cannot read exists of undefined`
- Since the repo is already loaded with lodash plugin, I have added get method
- Please change it to direct object reference according to your requirements


CC: @vinukumar-vs @rajeevsathish

Reply:
This is object null check. This is the good practice to have rather than using directly as `response.exists`. Loadash handles the null check of the `response` object.